astropy/utils/tests/test_decorators.py

```diff
@@ -687,12 +692,13 @@ def test_format_doc_stringInput_simple():
 
     docstring_fail = ""
 
-    # Raises an valueerror if input is empty
-    with pytest.raises(ValueError):
+    if sys.flags.optimize < 2:
```

Comment:
Might be less confusing if you move this one out into a different test function and use `@pytest.mark.skipif(sys.flags.optimize < 2, reason="NA for Python optimized mode")`

Reply:
done !